src/style.css

```diff
@@ -15,10 +15,18 @@ body {
 .wrapper {
   position: relative;
   width: 1440px;
-  min-height: 100vh;
+  height: 1379px;

```

Comment:
what is the purpose of hard coding the height value?

Reply:
the wrapper is not extending to the bottom of the page because 100vh fills just one screen

Reply:
ok, sound reasonable